clients/fides-js/src/lib/gpp/us-notices.ts

```diff
@@ -66,32 +69,27 @@ const deriveGppFieldRegion = ({
 export const setGppNoticesProvidedFromExperience = ({
   cmpApi,
   experience,
-  forceGpp,
 }: {
   cmpApi: CmpApi;
   experience: PrivacyExperience;
-  forceGpp: boolean;
 }) => {
-  const sectionsChanged = new Set<{ name: string; id: number }>();
+  const sectionsChanged = new Set<GPPSection>();
   const {
     privacy_notices: notices = [],
-    region,
+    region: experienceRegion,
     gpp_settings: gppSettings,
   } = experience;
+  const usApproach = gppSettings?.us_approach;
   const gppRegion = deriveGppFieldRegion({
-    experienceRegion: region,
-    usApproach: gppSettings?.us_approach,
+    experienceRegion,
+    usApproach,
   });
   const gppSection = FIDES_REGION_TO_GPP_SECTION[gppRegion];
 
-  if (!gppSection) {
-    if (
-      (forceGpp && !experience?.gpp_settings) ||
-      experience?.gpp_settings?.us_approach === GPPUSApproach.STATE
-    ) {
-      cmpApi.setApplicableSections([-1]);
```

Comment:
the logic for this was moved up where other applicable sections were being set, to avoid confusion